messaging/nats.go

```diff
@@ -1,20 +1,23 @@
 package messaging
 
 import (
+	"fmt"
 	"strings"
 
 	"github.com/nats-io/nats"
 	"github.com/sirupsen/logrus"
 
 	"github.com/rybit/nats_logrus_hook"
 
+	"github.com/netlify/netlify-commons/discovery"
 	"github.com/netlify/netlify-commons/tls"
 )
 
 type NatsConfig struct {
-	TLS         *tls.Config `mapstructure:"tls_conf"`
-	Servers     []string    `mapstructure:"servers"`
-	LogsSubject string      `mapstructure:"log_subject"`
+	TLS           *tls.Config `mapstructure:"tls_conf"`
+	DiscoveryName string      `mapstructure:"discovery_name"`
```

Comment:
add a `split_words:"true"` so that the envconfig would be something like `WHATEVER_NATS_DISCOVERY_NAME`.